### DISCUSSION TITLE
How to watch for file change and reload browser with vitejs

### DISCUSSION THREAD

**Discussion by axelilali** [wordplate/wordplate#356] [Q&A]
Hi !
First of all, THANK YOU !!! 🙌  I love Wordplate.

I'm not able to have hrm or automatically reload my theme when i update my css/js files.
Could you help ?

**Comment by vinkla** [comment]
I'm glad you like the project!

It should be able to live reload locally:

https://github.com/vinkla/wordplate/blob/303d8092621c1d61602aa984041a912581224c55/vite.config.js#L20-L24

I haven't tried HMR, if you find that the projects need to be updated for this purpose, please submit a pull request.
